src/plugin/indexer-elastic/ivy.xml

```diff
@@ -36,8 +36,8 @@
   </publications>
 
   <dependencies>
-    <dependency org="org.elasticsearch" name="elasticsearch" rev="5.3.0" conf="*->default"/>
-    <dependency org="org.elasticsearch.client" name="transport" rev="5.3.0"/>
+    <dependency org="org.elasticsearch.client" name="elasticsearch-rest-high-level-client" rev="7.3.0"/>
```

Comment:
The updated dependencies must be also "registered" in [indexer-elastic/plugin.xml](https://github.com/apache/nutch/blob/b5541455e0c6e2da4a8539ce61d2f05fc68f472c/src/plugin/indexer-elastic/plugin.xml). The description [hot to uprade ES](https://github.com/apache/nutch/blob/b5541455e0c6e2da4a8539ce61d2f05fc68f472c/src/plugin/indexer-elastic/howto_upgrade_es.txt) is somewhat outdated, better have a look at parse-tika for both the description and build-ivy.xml.

Reply:
Sure @sebastian-nagel will add the changes in plugin.xml and remove unnecessary dependencies.